### ISSUE TITLE
[Question] 

### ISSUE THREAD

**Issue by Qo12** [hardcore-sushi/DroidFS#137]
If I compress an encrypted volume as 7z and then move it to another operating system, can I reopen it?

**Comment by creeper451** [comment]
Yes you can but I recommend using something like "primitive ftp" from fdroid and using sftp protocol on an open source client like "filezilla" you could copy the volume directly into the other machine without having to compress and uncompress the folder.
